include/battle_util.h

```diff
@@ -195,7 +208,7 @@ u8 GetBattlerForBattleScript(u8 caseId);
 bool32 IsBattlerMarkedForControllerExec(u32 battler);
 void MarkBattlerForControllerExec(u32 battler);
 void MarkBattlerReceivedLinkData(u32 battler);
-const u8* CancelMultiTurnMoves(u32 battler);
+const u8* CancelMultiTurnMoves(u32 battler, enum SkyDropState skyDropState);
```

Comment:
```suggestion
const u8 *CancelMultiTurnMoves(u32 battler, enum SkyDropState skyDropState);
```
Not your fault, but may as wel clean it up while we're at it